### PR TITLE
Alua misc fix

### PR DESCRIPTION
**Pull request by lxbsz**

### CODE REVIEW

File: alua.c [hunk 1]
```diff
@@ -319,6 +323,8 @@ tcmu_get_tgt_port_grp(struct tcmu_device *dev, const char *name)
 	free(orig_str_val);
 	return group;
```
Comment (mikechristie) [inline alua.c]: These patches are nice for future kernel changes but this is not possible right now is it?

The kernel always returns one of the checked values? If not then, we are missing some checks for valid values.

**Comment by mikechristie** [comment]
Thanks.

File: alua.c (hunk 1 above)
Comment (mikechristie) [inline alua.c]: Oh yeah, I merged as is assuming it was for future proofing the code.

If there are other strings being returned let me know the kernel version and I can fix that.

Comment (lxbsz) [inline alua.c]: Yes, from the current kernel code, this usually should not happen till now.

**Pull request merged**